### PR TITLE
Officially drop support for Windows Vista

### PR DESCRIPTION
**Pull request by dscho**
The plan to do this had been announced already for a while.

### CODE REVIEW

File: requirements.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ This page lists the prerequisites required to run [Git for Windows](https://gitf
...
+Git for Windows requires Windows 7 Service Pack 1 or later. The last version to support Windows Vista was v2.37.1. The last version of Git for Windows to support Windows XP and Windows Server 2003 is [v2.10.0](https://github.com/git-for-windows/git/releases/tag/v2.10.0.windows.1).
 
 Why?
 
-Parts of Git are implemented in shell script, and Git for Windows runs those scripts via [MSYS2](https://msys2.github.io/)'s POSIX emulation layer, which in turn is based on the [Cygwin POSIX emulation layer](https://cygwin.com). Seeing as Windows XP and Windows Server 2003 are years past their official end of life, the Cygwin project ended their Herculean efforts to support those Windows versions.
+Parts of Git are implemented in shell script, and Git for Windows runs those scripts via [MSYS2](https://msys2.github.io/)'s POSIX emulation layer, which in turn is based on the [Cygwin POSIX emulation layer](https://cygwin.com). Seeing as Windows Vista, XP and Server 2003 are years past their official end of life, the Cygwin project ended their Herculean efforts to support those Windows versions.
```
Comment (rimrul) [inline requirements.md:11]: We could mention the Vista equivalent Server version, like we do with Windows Server 2003.

File: requirements.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ This page lists the prerequisites required to run [Git for Windows](https://gitf
 
 ## Windows version
 
-As of Git for Windows v2.10.1, Windows Vista or later are required. The last version of Git for Windows to support Windows XP and Windows Server 2003 is [v2.10.0](https://github.com/git-for-windows/git/releases/tag/v2.10.0.windows.1).
+Git for Windows requires Windows 7 Service Pack 1 or later. The last version to support Windows Vista was v2.37.1. The last version of Git for Windows to support Windows XP and Windows Server 2003 is [v2.10.0](https://github.com/git-for-windows/git/releases/tag/v2.10.0.windows.1).
```
[the remark this answers is not in the document]
Comment (dscho) [reply requirements.md]: Fair.

File: requirements.md (hunk 1 above)
Comment (dscho) [reply requirements.md]: Makes sense.

**Comment by dscho** [comment]
@rimrul the color-word inter-diff is:

image

**Comment by rimrul** [comment]
Looks good.

**Pull request merged**